getting-started-expressjs/README.md

```diff
@@ -0,0 +1,37 @@
+# Getting Started with Metaplex and Express.js
+
+This example sets up a new express app with Metaplex.
+The Express.js app works with only one route that accepts a mint address in the request and returns the NFT object from the SDK as the response.
+
+It requires [Node.js](https://nodejs.org/) to run.
+1. Clone the example
```

Comment:
If you look at the other examples, they tell you the steps to achieve this repo from scratch instead of having to clone the repo.

The readme should be more like a little tutorial to reach this example.

So step one could be:

**Create a new Node project**

```sh
mkdir getting-started-expressjs
cd getting-started-expressjs
npm init -y
```

Step two could be:

**Install the Metaplex and Solana SDKs**

```sh
npm install @metaplex-foundation/js-next @solana/web3.js
```

etc.

Reply:
Thanks @lorisleiva 
Could you please check it now, to see if it's okay now?

Reply:
I've commented on a new review. 🙂